app/views/filters/_form.html.erb

```diff
@@ -56,7 +63,7 @@
     <% if show_location_tab? %>
       <div class="tab-pane" id="locations">
         <% collection = User.current.locations.empty? ? Location : User.current.locations %>
-        <%= multiple_checkboxes f, :locations, f.object, collection, :label => _("Locations") %>
+        <%= multiple_checkboxes f, :locations, f.object, collection, {}, :label => _("Locations") %>
```

Comment:
this change only touches locations and on filter form making it the same as for organizations https://github.com/theforeman/foreman/blob/develop/app/views/filters/_form.html.erb#L66

the reason probably is that normally we use different data - https://github.com/theforeman/foreman/blob/develop/app/views/taxonomies/_loc_org_tabs.html.erb#L8 but that's separate issue, I'd suggest reporting it in redmine